pset-functions.js

```diff
@@ -91,10 +97,28 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+
+function wordCount(phrase) {
+  //spliting the phrase into words in an array
+  const phraseWords = phrase.split(" ")
+  //object to contain instances of a word
+  const wordMapObj = {}
+  //runs through object phraseWords
+  //and adds string of number = instances of word to object wordMapObj
+  phraseWords.forEach((word) => {
```

Comment:
@kyleligman nice approach, be careful with the syntax for your `if` statements. The current structure makes the code harder to read. See the lesson on conditionals:  https://jsd20190730.github.io/03-control-flow-and-loops/conditionals/ for the recommended approach to structuring conditionals.